src/routes/gallery.tsx

```diff
@@ -21,33 +21,118 @@ export const app = new Frog<HonoEnv>({})
 //       <Button.Link href="https://kodadot.xyz">kodadot</Button.Link>
 //     ],
 //   })
-// })  
+// })
+function UnverifiedImage({ imageUrl }: {imageUrl: string}) {
+  return (
+    <div
+      style={{
+        height: "100%",
+        width: "100%",
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#fff",
+        fontSize: 32,
+        fontWeight: 600,
+        position: "relative",
+      }}
+    >
+      <img
+        src={imageUrl}
+        width="100%"
+        height="100%"
+      />
+      <div
+        style={{
+          display: "flex",
+          gap: "10px",
+          position: "absolute",
+          marginTop: 40,
+          bottom: 20,
+          color: "red",
+        }}
+      >
+        Join <span style={{ color: "lime" }}>/koda</span> channel to mint
+      </div>
+    </div>
+  );
+}
+
 
 app.frame('/:chain/:id', async (c) => {
-// app.frame('/', async (c) => {
-// app.frame('/', async (c) => {
-  const { chain, id } = c.req.param()
-  const collection = await getContent('base', id, null)
-  const image = $purifyOne(collection.image, 'kodadot_beta')
-  const price = collection.price || MINT_PRICE
-
-  const label = `${collection.name} [${price} ETH]`
-  const target = `/${chain}/${id}/mint`
-  const action = `/${chain}/${id}/finish`
+  // app.frame('/', async (c) => {
+
+  const { chain, id } = c.req.param();
+  //MOCK
+  // const chain = 'ahp'
+  // const id = '106'
```

Comment:
```suggestion
  const { chain, id } = {
     chain: 'base',
     id: '0xd9a2c93ba2e9fae10fe762a42ee807bbf95764cc',
   } //c.req.param()
```

Reply:
The entire initial code actually remains unchanged. So it would work exactly the same as before by going to /base/${id}. The commented mocks were only used to make the new frame page and removed after that.

Reply:
> The entire initial code actually remains unchanged

Isnt it better to make a new route than make a lot of conditionals?

Reply:
My Idea was, since I wasn't able to test, to reduce how much I refactor things. But that's cleaner for sure. Should be straight forward to refactor. On it

Reply:
> I wasn't able to test


`0xd9a2c93ba2e9fae10fe762a42ee807bbf95764cc` this collection is mintable for  ` 0.0015 ETH` which can be reimbursed

Reply:
Alright. Will check it